Craigslist/Craigslist.Domain/Content/PagedList.css

```diff
@@ -0,0 +1,166 @@
+.pagination {
+  display: inline-block;
+  padding-left: 0;
+  margin: 20px 0;
+  border-radius: 4px;
+}
+
+.pagination > li {
+  display: inline;
```

Comment:
Why do you have css in domain!!!!!?????? O_o
And it's probably the same authogenerated css from paged list package.